tests/module/mobject/mobject/test_mobject.py

```diff
@@ -136,21 +136,19 @@ def test_mobject_dimensions_nested_mobjects():
 
 
 def test_mobject_dimensions_mobjects_with_no_points_are_at_origin():
```

Comment:
As the previous test was testing an unexpected behavior, it should have been linked to an issue.

Reply:
I agree.

Now, given that the unexpected behavior is fixed and empty Mobjects are no longer considered as "at the origin", it would be nice to change the name of this test in order to reflect that:
```suggestion
def test_mobject_dimensions_mobjects_with_no_points():
```